### PR TITLE
HW English task 1

### PR DESCRIPTION
**Pull request by i3oi3ka**
I did my first homework

### CODE REVIEW

**Review by Alex99905 — DISMISSED** [review]

File: homework/english/RudayV/Lecture1/grammar.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+##Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+Your mum working today?
+Why this boys are talking now?
+Where does he live?
+Does their mother cook very well?
+Lili is not rich, she does not to drive a Mercedes.
+Let´s go out. It is not raining now.
+I have to go now. It is geting dark
+A young man usually stand at the window.
```
Comment (olgaz-skiba) [inline homework/english/RudayV/Lecture1/grammar.md:10]: A young man usually **stands** at the window.

File: homework/english/RudayV/Lecture1/grammar.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+##Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+Your mum working today?
```
Comment (olgaz-skiba) [inline homework/english/RudayV/Lecture1/grammar.md:3]: Is your mum working today?

File: homework/english/RudayV/Lecture1/grammar.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+I have to go now. It is geting dark
+A young man usually stand at the window.
+Ann usually makes doll dresses herself.
+Is his wife having coffee at the moment?
+Let’s go for a walk. It is not raining.
+Look! The boys comming home.
+Does Maila like Chinese food?
+My grandmother does not working. She is on pension.
+What is your brother watching now?
+Where are you live?
```
Comment (olgaz-skiba) [inline homework/english/RudayV/Lecture1/grammar.md:18]: Where do you live?

**Review by olgaZskiba — DISMISSED** [review]
Igfm

**Review by i3oi3ka — COMMENTED** [review]
corrected mistakes

**Review by olgaZskiba — APPROVED** [review]
Igfm

File: homework/english/RudayV/Lecture1/grammar.md (hunk 1 above)
Comment (i3oi3ka) [reply homework/english/RudayV/Lecture1/grammar.md]: corrected mistakes

File: homework/english/RudayV/Lecture1/grammar.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+##Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+Your mum working today?
+Why this boys are talking now?
```
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md:4]: why are this boys talking..

File: homework/english/RudayV/Lecture1/grammar.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+Where does he live?
+Does their mother cook very well?
+Lili is not rich, she does not to drive a Mercedes.
```
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md:7]: does not drive (without "to")

File: homework/english/RudayV/Lecture1/grammar.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+Let´s go out. It is not raining now.
+I have to go now. It is geting dark
```
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md:9]: getTing

File: homework/english/RudayV/Lecture1/grammar.md (hunk 1 above)
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md]: standS

File: homework/english/RudayV/Lecture1/grammar.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+A young man usually stand at the window.
+Ann usually makes doll dresses herself.
+Is his wife having coffee at the moment?
+Let’s go for a walk. It is not raining.
+Look! The boys comming home.
```
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md:14]: boys are comming

**Review by Alex99905 — APPROVED** [review]

File: homework/english/RudayV/Lecture1/grammar.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+Does Maila like Chinese food?
+My grandmother does not working. She is on pension.
```
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md:16]: does not work

File: homework/english/RudayV/Lecture1/grammar.md (hunk 3 above)
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md]: where do you live?

File: homework/english/RudayV/Lecture1/grammar.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+What is your brother watching now?
+Where are you live?
+Do you play the piano well?
+Where is their brother swiming now?
```
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md:20]: swimMing

File: homework/english/RudayV/Lecture1/grammar.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+What are they eating at breakfast?
+Are they swiming at the moment?
```
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md:22]: swimMing

File: homework/english/RudayV/Lecture1/grammar.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+Mike does not wear a tie today.
```
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md:23]: isn't wearing

File: homework/english/RudayV/Lecture1/grammar.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+Maila does not watch TV every morning.
+Listen! Blanca is singing in the bathroom.
+Bea sometimes reads comics.
+
+##Найдите ошибки и исправьте их.
+
+We are not going to school today.
+What are you doing after school?
+At the moment Peter is working in Russia.
+Does he have a new car?
```
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md:33]: has

File: homework/english/RudayV/Lecture1/grammar.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+He never wears a hat.
+He doesn’t like black coffee.
+We are haveing a good time.
```
Comment (Littledrop777) [inline homework/english/RudayV/Lecture1/grammar.md:36]: having

**Review by Alex99905 — APPROVED** [review]

**Review by Svetlana101010 — APPROVED** [review]

**Pull request merged**